flask_restplus_patched/parameters.py

```diff
@@ -72,3 +76,23 @@ def __init__(self, *args, **kwargs):
             self.fields['op'].validators + [validate.OneOf(self.OPERATION_CHOICES)]
         self.fields['path'].validators = \
             self.fields['path'].validators + [validate.OneOf(self.PATH_CHOICES)]
+
+    @validates_schema
+    def validate_patch_structure(self, data):
+        """
+        Common validation of PATCH structure
+
+        Provide check that 'value' present in all operations expect it.
+
+        Provide check if 'path' is present. 'path' can be absent if provided
+        without '/' at the start. Supposed that if 'path' is present than it
+        is prepended with '/'.
+        Removing '/' in the beginning to simplify usage in resource.
+        """
+        if data['op'] not in self.NO_VALUE_OPERATIONS and 'value' not in data:
+            raise ValidationError('value is required')
+
+        if 'path' not in data:
```

Comment:
The thins is in case that `path` provided without `/` we have no `path` in `data` but we have all the rest stuff - `value` and `op` if provided. And there is no error without this check. 

And if `path` is in `data` then surely it has `/` in the beginning.

Does it make sense?